diff -urN PDL-2.4.0.org/Graphics/Karma/Makefile.PL PDL-2.4.0/Graphics/Karma/Makefile.PL
--- PDL-2.4.0.org/Graphics/Karma/Makefile.PL	2003-12-30 14:54:23.403380904 +0100
+++ PDL-2.4.0/Graphics/Karma/Makefile.PL	2003-12-30 15:48:19.268102181 +0100
@@ -85,7 +85,7 @@
 @pack = (["karma.pd",Karma,PDL::Graphics::Karma]);
 
 %hash = pdlpp_stdargs(@::pack);
-$hash{INC}         .= " -I$karmabase/include $mallocinc";
+$hash{INC}         .= " -I/usr/include/karma -I$karmabase/include $mallocinc";
 $hash{LIBS}[0]     .=' -L'. $karmabase."/lib" .' -lkarma -lkarmagraphics -L/usr/X11R6/lib -lkarmaX11 -lX11 -lXext -lm ' . $malloclib;
 # change here approriately for other platforms !
 if ($^O =~ 'irix' or $^O eq 'dec_osf') { $rflag = '-rpath ' } else { $rflag = '-R' }
diff -urN PDL-2.4.0.org/perldl.conf PDL-2.4.0/perldl.conf
--- PDL-2.4.0.org/perldl.conf	2003-12-30 15:27:30.234521263 +0100
+++ PDL-2.4.0/perldl.conf	2003-12-30 15:48:46.554409982 +0100
@@ -84,8 +84,8 @@
 #	OPENGL_DEFINE => undef,
 # Whether or not to build the Karma interface module
 
-        WITH_KARMA => undef,    # Leave it up to PDL to decide
-        WHERE_KARMA => 0,   # you may explicitly specify directory location
+        WITH_KARMA => 1,    # Leave it up to PDL to decide
+        WHERE_KARMA => "/usr/lib/karma",   # you may explicitly specify directory location
 
 #
 ## Whether or not to build the PLplot interface module
