diff -ur PDL-2.4.3/Lib/GSL/Makefile.PL PDL-2.4.3-gsl/Lib/GSL/Makefile.PL
--- PDL-2.4.3/Lib/GSL/Makefile.PL	2008-04-08 11:00:19.000000000 +0200
+++ PDL-2.4.3-gsl/Lib/GSL/Makefile.PL	2008-04-08 10:59:34.000000000 +0200
@@ -41,7 +41,10 @@
   warn "\tno GSL version info found (gsl-config not installed?)\n\n";
   $version = 'UNKNOWN VERSION';
 }
-if ($version =~ /UNKNOWN VERSION/ || $version lt "$MINVERSION") {
+my $MINVERMAJ = "1";
+my $MINVERMIN = "3";
+my ($vmaj,$vmin) = split(/\./,$version);
+if ($version =~ /UNKNOWN VERSION/ || $vmaj lt $MINVERMAJ || ($vmaj eq $MINVERMA && $vmin lt $MINVERMIN)) {
   warn "\n   Not building GSL modules: GSL version $version found, need at least $MINVERSION\n\n";
   write_dummy_make("Not building GSL modules: GSL version $version found, but need at least $MINVERSION");
 } else {
