diff -urN PDL-2.4.0.org/perldl.conf PDL-2.4.0/perldl.conf
--- PDL-2.4.0.org/perldl.conf	2003-12-30 14:54:25.663910257 +0100
+++ PDL-2.4.0/perldl.conf	2003-12-30 15:14:41.363631658 +0100
@@ -58,15 +58,15 @@
 # false -> skip build of PDL::Graphics:::TriD
 # undef -> let PDL build decide based on dependencies present
 #
-        WITH_3D => undef,
+        WITH_3D => 1,
 
 # Build Graphics/TriD using Perl OpenGL
 #
 # true -> use new Perl OpenGL bindings
 # false -> use legacy, deprecated X11 only bindings
 # undef -> let PDL build decide (check if Perl OpenGL is present)
 #
-        USE_POGL => undef,
+        USE_POGL => 1,
 #       USE_POGL => 0,
 #
         POGL_VERSION => 0.63,           # minimum compatible OpenGL version
@@ -110,7 +110,7 @@
 #    about curses to fix it 
 #  - 1 builds, 0 or undef does not build
 #
-        WITH_IO_BROWSER => 0,
+        WITH_IO_BROWSER => 1,
 
 # Quiet Astro::FITS::Header warnings for PDL build process by default
 # Eventually would be better to set undef by default, and have the
