diff -urN PDL-2.4.0.org/perldl.conf PDL-2.4.0/perldl.conf
--- PDL-2.4.0.org/perldl.conf	2003-12-30 14:54:25.663910257 +0100
+++ PDL-2.4.0/perldl.conf	2003-12-30 15:14:41.363631658 +0100
@@ -58,16 +58,19 @@
 # There are problems with the build on OS-X, so we turn it off by default
 # for such systems
 #
-	WITH_3D => $^O eq "darwin" ? 0 : undef,
+	WITH_3D => 1,
 
 #
 #  For Mesa 3.2
 #
 #	OPENGL_LIBS => '-L/usr/local/Mesa/lib -lGL -lGLU -lXext -lX11',
+	OPENGL_LIBS => '-L/usr/lib -lGL -lGLU -lXext -lX11',
 #  Mesa need -DGL_GLEXT_LEGACY for glPolygonOffsetEXT amoung others
 #
 #	OPENGL_DEFINE => '-DGL_GLEXT_LEGACY',
 # 	OPENGL_INC => '-I/usr/include/GL/',
+	OPENGL_DEFINE => '-DGL_GLEXT_LEGACY',
+	OPENGL_INC => '-I/usr/include',
 #
 # 
 # For SGI GL
@@ -76,7 +79,7 @@
 # Whether or not to build the Karma interface module
 
         WITH_KARMA => undef,    # Leave it up to PDL to decide
-        WHERE_KARMA => undef,   # you may explicitly specify directory location
+        WHERE_KARMA => 0,   # you may explicitly specify directory location
 
 #
 ## Whether or not to build the PLplot interface module
