--- PDL-2.2.1/Lib/GSL/Makefile.PL.orig	Fri Apr 21 08:23:55 2000
+++ PDL-2.2.1/Lib/GSL/Makefile.PL	Thu May 17 22:24:00 2001
@@ -30,14 +30,14 @@
 
 	# Look for GSL libs:
 	# default locations:
-	@GSL_lib_locations = ('/lib/libgsl.a','/usr/lib/libgsl.a','/usr/local/lib/libgsl.a'); 
+	@GSL_lib_locations = ('/lib/libgsl.so','/usr/lib/libgsl.so','/usr/local/lib/libgsl.so'); 
 
 	# get locations from perldl.conf, if specified there:
 	@GSL_lib_locations = @{$PDL_CONFIG{GSL_LIBS}} if( defined $PDL_CONFIG{GSL_LIBS} ); 
 
 	foreach $dir (@GSL_lib_locations) {
-		if (-e "$dir/libgsl.a") {
-			$gsl = "$dir/libgsl.a";
+		if (-e "$dir/libgsl.so") {
+			$gsl = "$dir/libgsl.so";
 			$gsldir = $dir;
 			$donot = 0;
 			print "${gsl} found, hope is version >= 0.4.1 for full functionality!\n";
