diff -urNp -x '*.orig' PDL-2.018.org/Makefile.PL PDL-2.018/Makefile.PL
--- PDL-2.018.org/Makefile.PL	2021-05-09 00:22:58.841780681 +0200
+++ PDL-2.018/Makefile.PL	2021-05-09 00:22:58.928448347 +0200
@@ -557,6 +557,11 @@ doc_vendor_install ::
 %HTML%	$(NOECHO) $(ECHO) "doc_vendor_install: Building PDL documentation web pages ..."
 %HTML%	$(NOECHO) $(PERL) Doc$(DFSEP)mkhtmldoc.pl "$(DESTDIR)$(INSTALLVENDORARCH)$(DFSEP)PDL"
 
+doc_vendor_install ::
+	@echo "doc_vendor_install: Building PDL documentation database ..."
+	@$(PERL) -Mblib Doc/scantree.pl $(DESTINSTALLVENDORARCH)
+	@$(PERL) Doc/mkhtmldoc.pl $(DESTINSTALLVENDORARCH)/PDL
+	
 doc_perl_install ::
 	$(NOECHO) $(ECHO) "doc_perl_install: Building PDL documentation database ..."
 	$(NOECHO) $(PERLRUNINST) Doc$(DFSEP)scantree.pl "$(DESTDIR)$(INSTALLARCHLIB)"
