diff -urNp -x '*.orig' PDL-2.018.org/perldl.conf PDL-2.018/perldl.conf
--- PDL-2.018.org/perldl.conf	2017-01-14 22:13:33.000000000 +0100
+++ PDL-2.018/perldl.conf	2021-05-09 00:22:58.271774104 +0200
@@ -107,7 +107,7 @@
 # false -> skip build of PDL::Graphics:::TriD
 # undef -> let PDL build decide based on dependencies present
 #
-        WITH_3D => undef,
+        WITH_3D => 1,
 
 # Build Graphics/TriD using Perl OpenGL
 #
@@ -115,7 +115,7 @@
 # false -> use legacy, deprecated X11 only bindings
 # undef -> let PDL build decide (check if Perl OpenGL is present)
 #
-        USE_POGL => undef,
+        USE_POGL => 1,
 #       USE_POGL => 0,
 #
         POGL_VERSION => 0.6702,           # minimum compatible OpenGL version
@@ -189,7 +189,7 @@
 # Do we build PDL::IO::Browser?
 # default -> do not build
 #
-        WITH_IO_BROWSER => 0,
+        WITH_IO_BROWSER => 1,
 
 # Quiet Astro::FITS::Header warnings for PDL build process by default
 # Eventually would be better to set undef by default, and have the
