diff -Nur PDL-2.1.2cvs20001124.orig/perldl.conf PDL-2.1.2cvs20001124/perldl.conf
--- PDL-2.1.2cvs20001124.orig/perldl.conf	Mon Sep 25 22:59:31 2000
+++ PDL-2.1.2cvs20001124/perldl.conf	Sat Nov 25 17:58:04 2000
@@ -45,16 +45,16 @@
 #	BADVAL_USENAN => 1,
 
 # Try to build Graphics/TriD
-	WITH_3D => undef,
+	WITH_3D => 1,
 #
 #  For Mesa 3.2
 #
-#	OPENGL_LIBS => '-L/usr/local/Mesa/lib -lGL -lGLU -lXext -lX11',
+	OPENGL_LIBS => '-L/usr/X11R6/lib -lGL -lGLU -lXext -lX11',
 #
 #  Mesa need -DGL_GLEXT_LEGACY for glPolygonOffsetEXT amoung others
 #
-#	OPENGL_DEFINE => '-DGL_GLEXT_LEGACY',
-# 	OPENGL_INC => '-I/usr/local/Mesa/include',
+	OPENGL_DEFINE => '-DGL_GLEXT_LEGACY',
+ 	OPENGL_INC => '-I/usr/X11R6/include',
 #
 # 
 # For SGI GL
@@ -64,13 +64,13 @@
 #	OPENGL_DEFINE => '',
 #
 # Let perl try to figure it out
-	OPENGL_LIBS => undef,
- 	OPENGL_INC => undef,
-	OPENGL_DEFINE => undef,
+#	OPENGL_LIBS => undef,
+# 	OPENGL_INC => undef,
+#	OPENGL_DEFINE => undef,
 # Whether or not to build the Karma interface module
 
         WITH_KARMA => undef,    # Leave it up to PDL to decide
-        WHERE_KARMA => undef,   # you may explicitly specify directory location
+        WHERE_KARMA => 0,   # you may explicitly specify directory location
 	
 # Whether or not to build the PDL::Slatec module
 # 0 -> don't use
